src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -337,4 +400,50 @@ protected function getGridActions()
                     ->setIcon('storage')
             );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getBulkActions()
+    {
+        return (new BulkActionCollection())
+            ->add(
+                (new AjaxBulkAction('enable_selection_ajax'))
+                    ->setName($this->trans('Activate selection', [], 'Admin.Actions'))
+                    ->setOptions([
+                        'ajax_route' => 'admin_products_v2_active_ajax',
+                        'modal_title' => $this->trans('Activating products', [], 'Admin.Actions'),
```

Comment:
```suggestion
                        'modal_title' => $this->trans('Activating products', [], 'Admin.Catalog.Notification'),
```
It's better to use the existing string instead of creating a new one.